features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/rolesandpermissions/RolesAndPermissionsFlowNode.kt

```diff
@@ -57,6 +60,16 @@ class RolesAndPermissionsFlowNode @AssistedInject constructor(
         data class ChangeRoomPermissions(val section: ChangeRoomPermissionsSection) : NavTarget
     }
 
+    override fun onBuilt() {
+        super.onBuilt()
+        whenChildAttached { lifecycle, node: ChangeRoomMemberRolesEntryPoint.NodeProxy ->
+            lifecycle.coroutineScope.launch {
+                node.waitForRoleChanged()
+                backstack.pop()
+            }
+        }
+    }
+
```

Comment:
I don't think we need this here, we don't allow demoting yourself as an owner in this screen or any of their sub-screens.